jest.config.cjs

```diff
@@ -200,6 +206,12 @@ const customJestConfig = {
       lines: 72,
       statements: 72,
     },
+    "src/app/components/client/toolbar/UserMenu.tsx": {
```

Comment:
This was only meant for transitioning existing code - the idea was that new code is either covered by unit tests, or [marked as ignored for coverage](https://github.com/istanbuljs/v8-to-istanbul#ignoring-uncovered-lines). Any reason this code shouldn't be unit tested?